community/event-checklist.md

```diff
@@ -0,0 +1,95 @@
+
+✔️ Refer [kossiitkgp/mistakes](https://github.com/kossiitkgp/mistakes) repository to get an idea of what sought of things can go wrong.
+
+✔️ It is a good idea to record the event(offline/online) for the sake of posterity.
+
+✔️ Make a Github repo for the workshop with name as event name and year which contains - installation instructions, slides, references, recording and others. (For example refer - [1](https://github.com/kossiitkgp/git-and-github-workshop-2020), [2](https://github.com/kossiitkgp/REST-APIs-in-Flask-Workshop-2021), [3](https://github.com/kossiitkgp/Linux-Shell-Vim-Workshop-2021))
+
+## 1.1 Internal
+
+- Keep in touch regularly with other team mates on updates regarding the event. Regular communication on #events channel is suggested.
+- Meet regularly to discuss stuff. It is advisable to send mail invites even for a smaller meet, to avoid procrastination.
+
+- **Making the workshop interactive**: It is always a good thing to keep the workshop interactive. If the workshop is conducted offline, we can have members of KOSS help people personally with hands on during the workshop and clear doubts more personally. But during an online workshop, this is not possible. So for sake of interaction, the speakers can encourage the audience to share the screen to show the work.  This acts as validation and also helps the rest of the audience. Asking the audience to share their screen in case of difficulties should also be encouraged. However note that,**do not force the audience to share their screen for sake of interaction. It has be voluntary and audience should not feel uncomfortable through such actions.**
+
+# 2. Technicalities - Workshop content
+
+## 2.1 How to decide the Topic for the workshop
+
+- Make sure that the topic adds value to the community. The topic doesn't have to be trendy or in vogue.
+- Atleast a couple of members at KOSS should have enough practical experience to speak about the topic
+
+## 2.2 How to decide the content of the workshop
+
+Make sure that the content adds a practical value to the audience. It might be tempting to add bits of cool jargon or theoretically detailed stuff to impress. But it is not recommended. For every slide, ask- Is this something that adds practical value. Will knowing or not knowing a particular thing affect the practical understanding of the topic. For example, consider explaining the idea of Merkel Tree in git, it might not be necessary to include the explanation of Merkel tree. As someone getting started with git, it is more important to practically understand staging area, branching, commits, sending a PR, pushing and pulling. When we get into details like Merkel Tree, audience might lose track of the event and lose interest. However, for the curious audience, it is a good idea to include such things at the end of the workshop, and provide further resources to explore more. 
```

Comment:
Exactly, it is important that we deliver what is important practically, rather than just delivering everything we know about the topic. I first heard about it in a podcast - https://www.scienceasylum.com/podcast/ATOTS001.mp3

"While the end of the semester, students just remember half of the content, and we should make sure that they remember the right half"

Since our time is limited, we should choose things of practical importance!